source/crud/tests/unified/estimatedDocumentCount.yml

```diff
@@ -135,3 +139,43 @@ tests:
                 count: *collection0Name
               commandName: count
               databaseName: *database0Name
+
+  - description: "estimatedDocumentCount works correctly on views"
+    # viewOn option was added to the create command in 3.4
+    runOnRequirements:
+      - minServerVersion: "3.4.0"
+    operations:
+      - name: dropCollection
+        object: *database0
+        arguments:
+          collection: *collection0ViewName
+      - name: createCollection
```

Comment:
For legacy reasons the Python driver issues a listCollections command before running createCollection. Could we add a `ignoreCommandMonitoringEvents: ["listCollections"]` to `client0` or use runCommand instead? Otherwise the test will fail during event assertion and I can't think of a better way to workaround it on the python side.

Reply:
How does Python pass the collection management tests (e.g. ["insertMany with duplicate ids"](https://github.com/mongodb/specifications/blob/master/source/collection-management/tests/timeseries-collection.yml#L63)), which use `createCollection` without ignoring those events? If possible, I'd rather not introduce a Python-specific exception to the spec tests if it can be avoided.

Reply:
We manually inserted the listCollections events into those files which is unfortunate because we have to maintain those edits every-time we resync the specs. But I agree with you. I'll find a workaround in python.